docs/source/ko/_toctree.yml

```
@@ -49,8 +49,8 @@
     title: 자연어처리
     isExpanded: false
   - sections:
-      - local: in_translation
-        title: (번역중) Audio classification
+      - local: tasks/audio_classification
+        title: 오디오 분류
       - local: tasks/asr
         title: 자동 음성 인식
     title: (번역중) 오디오
```

```suggestion
     title: 오디오
```

이 섹션은 완료되었으므로 번역중을 빼셔도 될 것 같아요. 좋은 번역 감사합니다, @gabrielwithappy 님!